README.md

```diff
@@ -1,56 +1,73 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Covid19 Tracker Web app
 
+> A web app built with Covid-19-API that displays Covid 19 cases, deaths and recovery per country in the entire World
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS, JavaScript
+- React
+- Redux
+- WebPack 
+- Covid19 API
 
-## Live Demo (if available)
+## Live Demo 
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo](https://634b18b7735def69fb36e1ca--musical-sunburst-e30cc3.netlify.app/)
 
 
-## Getting Started
+## Video Demo 
+
+[Video Demo](https://drive.google.com/file/d/15cvM6YNoK8lCaV5BIxsvKP2Tg24AxtFp/view?usp=sharing)
```

Comment:
- [ ] Good job with the project presentation, however, you missed some steps in your video presentation. Your video presentation only showed the functional aspect of your app and failed to highlight any portion of your code. Kindly ensure that the video contains a highlight about a piece of the code that you find interesting or that you were proud of making.